account_commission_settlement_processing_fee/models/commission.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2022 Quartile Limited
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class Commission(models.Model):
+    _inherit = "commission"
+
+    calculate_listing_fee = fields.Boolean()
+    listing_fee_product_id = fields.Many2one(
+        comodel_name="product.product", domain="[('type', '=', 'service')]"
+    )
```

Comment:
```suggestion
```
Please move this field to res.partner.  We would like to apply the rate according to the service plan the seller subscribes to.